.github/workflows/ci.yaml

```diff
@@ -0,0 +1,49 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  build_test:
+    name: Build and Test
+    strategy:
+      matrix:
+        go: ['1.12', '1.13', '1.14', '1.15', '1.16']
+    runs-on: ubuntu-latest
+    container:
+      image: ubuntu
+      volumes:
+      - /etc/ssl/certs:/etc/ssl/certs
+    services:
+      postgres:
+        image: ubuntu/postgres
+        env:
+          POSTGRES_PASSWORD: shei4eeJoo3Pee0rahng
```

Comment:
I hope this was a password generated for this purpose and not reused from elsewhere...

To avoid confusion there I'd either use Github Secrets for this (https://docs.github.com/en/actions/reference/encrypted-secrets) or use something simple like "password" if it's a test throwaway database (like https://docs.github.com/en/actions/guides/creating-postgresql-service-containers).

Reply:
It certainly was generated for this purpose.